spec/controllers/sign_up/completions_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe SignUp::CompletionsController do
+  describe '#show' do
+    it 'tracks page visit' do
+      stub_sign_in
+      stub_analytics
+      allow(@analytics).to receive(:track_event).with(Analytics::USER_REGISTRATION_AGENCY_HANDOFF_PAGE_VISIT, { loa3: nil, service_provider_name: nil })
+
```

Comment:
I don't mind the increased specificity, but removing is cool w me too. Removed at fbf10ce